Packages/com.unity.inputsystem/InputSystem/Actions/InputActionSetupExtensions.cs

```diff
@@ -848,6 +848,26 @@ public static void AddControlScheme(this InputActionAsset asset, InputControlSch
         /// <returns>Syntax to allow providing additional configuration for the newly added control scheme.</returns>
         /// <exception cref="ArgumentNullException"><paramref name="asset"/> is <c>null</c> -or- <paramref name="name"/>
         /// is <c>null</c> or empty.</exception>
+        /// <remarks>
+        /// <example>
+        /// <code>
+        /// // Create an .inputactions asset.
+        /// var asset = ScriptableObject.CreateInstance&lt;InputActionAsset&gt;();
```

Comment:
Same thing as above. Code samples can be in a script and linked so that they don't rot over time.